config/boards/fxblox-rk1.wip

```diff
@@ -0,0 +1,82 @@
+# Rockchip RK3588 SoC octa core 8/16/32GB SoC USB-C DP NvME SATA M2
+BOARD_NAME="FxBlox RK1"
+BOARDFAMILY="rockchip-rk3588"
```

Comment:
`wip` require a maintainer.

Reply:
I want to be the maintainer of this board. What do I need to do?

Reply:
@igorpecovnik

Reply:
> What do I need to do?

Read / follow https://docs.armbian.com/Board_Maintainers_Procedures_and_Guidelines/ 

Here https://www.armbian.com/maintainer-registry/  put some of your contact data
(put _fxblox-rk1_ into field "Maintaining") ... and the rest is semi automated.

Reply:
@igorpecovnik, I registered at  https://www.armbian.com/maintainer-registry/